libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/service/VerifiableCredentialService.java

```diff
@@ -102,7 +102,11 @@ public void deleteVcStoreItems(String userId, Boolean isUserInitiated) {
             LOGGER.info(message);
         }
         for (VcStoreItem item : vcStoreItems) {
-            dataStore.delete(item.getUserId(), item.getCredentialIssuer());
+            deleteVcItem(item.getUserId(), item.getCredentialIssuer());
         }
     }
+
+    public void deleteVcItem(String userId, String criId) {
```

Comment:
Should this be `deleteVcStoreItem` for consistency? (sorry :) )

Reply:
Don't be!